api_app/script_analyzers/observable_analyzers/urlscan.py

```diff
@@ -0,0 +1,81 @@
+import requests
+import time
+import json
+from api_app.exceptions import AnalyzerRunException
+from api_app.script_analyzers import classes
+from intel_owl import secrets
+
+
+class UrlScan(classes.ObservableAnalyzer):
+    base_url: str = "https://urlscan.io/"
+
+    def set_config(self, additional_config_params):
+        self.analysis_type = additional_config_params.get("urlscan_analysis", "search")
+        api_key_name = additional_config_params.get("api_key_name", "URLSCAN_KEY")
```

Comment:
URLSCAN_KEY still missing in the env_file_template

Reply:
Sorry about this! I keep changing it in the env_file_app but forget to do the same in the template. Have fixed this and pushed again!